oshc/main/static/main/js/script.js

```diff
@@ -5,4 +5,68 @@ $(document).ready(function() {
       scrollTop: 0
     }, 1000);
   });
+  //  code for PASSWORD STRENGTH METER
+  $(".strength-wrapper").css("visibility", "hidden"); //hide the strength meter initially
+  console.log("checker2");
+  $("#id_password1").bind("keyup", function() {
+    if ($(this).val().length == 0) {
```

Comment:
@vaibhavsingh97  I didn't change the PR code somehow it happened and this is working fine locally.